Snake/BusinessLayer/Engine.cs

```diff
@@ -30,17 +26,6 @@ public class Engine
         private Food food;
         private FoodGenerator foodGenerator;
 
-        private enum gameMode
-        {
-            basic,
-        }
-        public enum snakeAction
-        {
-            eat,
-            die,
-            move
-        }
-
         gameMode currentMode = gameMode.basic;
```

Comment:
Andrew has implemented the menu, including selecting the game mode. The game mode should come from the UI and be hard coded.